chaoslib/litmus/cpu_hog/pod_cpu_hog.yml

```diff
@@ -0,0 +1,144 @@
+---
+- block: 
+
+    - block: 
+
+      - name: Select the app pod
+        shell: >
+          kubectl get pod -l {{ app_label }} -n {{ app_ns }}
+          -o=custom-columns=:metadata.name --no-headers
+          | shuf | head -1 
+        args:
+          executable: /bin/bash
+        register: app_pod_name
+
+      - name: Record app pod name
+        set_fact:
+          app_pod: "{{ app_pod_name.stdout }}"
+
+      when: "app_pod is not defined"
```

Comment:
Can we  this check  `when: "app_pod is not defined or app_pod == '' "` ?

Reply:
done!